serverless/cmd/integrate/main.go

```diff
@@ -128,15 +127,24 @@ func main() {
 	}
 }
 
+func getKeyFile(path string) (key string, err error) {
+	k, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "", fmt.Errorf("failed to read key file: %q", err)
```

Comment:
nit: Generally a good habit to get into is using `%w` when formatting errors - this allows the caller to (optionally) dig into the underlying reason for the failure (see https://golang.org/pkg/errors/ for a bit more detail on [un]wrapping).

(if you decide to change, can you sweep through the code and change the others too to keep it consistent?)

Reply:
Done